frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -54,6 +55,58 @@ import {
 import './ocs-storage-class-form.scss';
 import { GUARDED_FEATURES } from '../../features';
 
+export const CephFsNameComponent: React.FC<ProvisionerProps> = ({ onParamChange }) => {
+  const { t } = useTranslation();
+
+  const [externalFsName, setExternalFsName] = React.useState('');
+
+  const [cephClusters, loaded, loadError] = useK8sWatchResource<CephClusterKind[]>(
+    cephClusterResource,
+  );
+
+  const onExternalFsNameChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const name = e.currentTarget.value;
+    setExternalFsName(name);
+    onParamChange(name);
+  };
+
+  React.useEffect(() => {
+    if (!loaded || loadError) onParamChange('');
+    if (loaded && !loadError && cephClusters[0]?.metadata.name === CEPH_INTERNAL_CR_NAME) {
+      onParamChange(OCS_STORAGECLUSTER_CEPHFILESYSTEM);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [cephClusters, loaded, loadError]);
+
+  if (loaded && !loadError) {
+    return (
+      <div className="form-group">
+        <label className="co-required">{t('ceph-storage-plugin~Filesystem Name')}</label>
+        <input
+          className="pf-c-form-control"
+          type="text"
+          value={
+            cephClusters[0]?.metadata.name === CEPH_INTERNAL_CR_NAME
+              ? OCS_STORAGECLUSTER_CEPHFILESYSTEM
+              : externalFsName
```

Comment:
```suggestion
`${clusterName}-filesystem`
```
Adapt accordingly.

Reply:
We do not have any default fs name is case of external-mode. So, I have initialised it as '' (empty string) in the updated code, instead of  `${clusterName}-filesystem`

Reply:
Or should I use the fs name that we use in `ocs-external-storagecluster-cephfs` by default, but, allow user to make changes if they want ?